### PR TITLE
refactor: use new strip packages

### PR DESCRIPTION
**Pull request by BarryThePenguin**

### CODE REVIEW

File: src/components/use-stripe.tsx [hunk 1]
```diff
@@ -1,62 +0,0 @@
...
-      <HookProvider>{children}</HookProvider>
-    </ElementsInner>
-  );
-};
-
-Elements.propTypes = {
-  children: PropTypes.node.isRequired
-};
-
-export const useStripe = (): TStripeContext => useContext(StripeContext);
```
Comment (BarryThePenguin) [inline src/components/use-stripe.tsx]: 👋🏻

Comment (readeral) [reply src/components/use-stripe.tsx]: lol

**Comment by readeral** [comment]
What's the deal with having had the script in the footer, moving it to the head?

**Comment by BarryThePenguin** [comment]
> What's the deal with having had the script in the footer, moving it to the head?

I think Stripe just recommends putting it in the head

https://github.com/stripe/stripe-js/blob/master/README.md#manually-include-the-script-tag

**Pull request merged**